### PR TITLE
Update index.tsx

### PR DESCRIPTION
**Pull request by mehtavishwa30**

### CODE REVIEW

File: src/pages/index.tsx [hunk 1]
```diff
@@ -115,12 +115,12 @@ export default function Home({
               <Box>
                 <Text lineHeight="180%" fontSize="md" mb={4}>
                   <Link
-                    href="/blog/2024/05/21/solidity-0.8.26-release-announcement/"
+                    href="https://soliditylang.org/blog/2024/09/04/solidity-0.8.27-release-announcement"
```
Comment (nikola-matic) [inline src/pages/index.tsx:118]: You don't need the full URL, `/blog/2024/09/04...` will suffice.

Comment (r0qs) [inline src/pages/index.tsx:118]:
```suggestion
                    href="/blog/2024/09/04/solidity-0.8.27-release-announcement/"
```
The link should be relative.

Comment (r0qs) [reply src/pages/index.tsx:118]: Noticed that Nikola already replied, so resolving this.

Comment (mehtavishwa30) [reply src/pages/index.tsx:118]: oh you're right. I think @r0qs suggested the same.

**Review by r0qs — APPROVED** [review]

**Pull request merged**